sdk/attestation/attestation/sample.env

```diff
@@ -0,0 +1,9 @@
+AAD_ATTESTATION_URL="<AAD attestation URL>"
+ISOLATED_ATTESTATION_URL="<Isolated attestaion URL>"
+policySigningCertificate2=""
+isolatedSigningCertificate=""
+policySigningCertificate1=""
+policySigningCertificate0=""
```

Comment:
Sorry, why are these not using snake-casing?

Reply:
Honest answer? Because I wrote this for .Net back in October and was following .Net naming conventions. I then simply copied the test-resources-pre.ps1 script into the different languages and didn't update the output properties to reflect the target language.

Reply:
I don't mind, can defer to later if we want to bring consistency.